python/django/v1/ampli_app/.env.example

```diff
@@ -0,0 +1,2 @@
+API_KEY=ABCD
```

Comment:
Can you please name these to match the other examples https://github.com/amplitude/ampli-examples/blob/main/node/typescript/.env.example